Doc/whatsnew/3.9.rst

```diff
@@ -685,6 +685,10 @@ in nanoseconds.  The benchmarks were measured on an
 running the macOS 64-bit builds found at
 `python.org <https://www.python.org/downloads/mac-osx/>`_.
 
+* A number of Python builtins (:class:`range`, :class:`tuple`, :class:`set`, :class:`frozenset`, :class:`list`, :class:`dict`)
```

Comment:
IMO would be better if  we are more specific, something like: 

```The Python builtins: :class:`range`.....```

What about?